notebook/static/base/js/i18n.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// Module to handle i18n ( Internationalization ) and translated UI
+
+define([
+    'jed',
+    'moment',
+    'json!../../../i18n/nbjs.json',
+    'json!../../../i18n/de/LC_MESSAGES/nbjs.json',
```

Comment:
Yes, there is, but it usually requires writing a JS plugin in order to do it. I've run into a similar problem in other projects I have worked on.  What you really want here is a synchronous load, which is kind of a no-no in JS these days.   It can be done, and I'm willing to contribute it if/when people actually need a real translation done.